src/ResourceManagement/ServiceFabric/Microsoft.Azure.Management.ServiceFabric/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("ServiceFabric")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("ServiceFabric")]
+[assembly: AssemblyCopyright("Copyright �  2017")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+// Version information for an assembly consists of the following four values:
+//
+//      Major Version
+//      Minor Version 
+//      Build Number
+//      Revision
+//
+// You can specify all the values or you can default the Build and Revision Numbers 
+// by using the '*' as shown below:
+// [assembly: AssemblyVersion("1.0.*")]
+[assembly: AssemblyVersion("0.1.0.0")]
+[assembly: AssemblyFileVersion("0.1.0.0")]
```

Comment:
@chingchenmsft both `AssemblyVersion` and `AssemblyFileVersion` should be 1.0.0.0

`AssemblyFileVersion` will always match the version found in `project.json`, and `AssemblyVersion` will never be updated, unless you bump the major version of the package, in which case the `AssemblyVersion` should also have its major version bumped